<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusForm</class>
 <widget class="QDialog" name="StatusForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;URW Gothic L&quot;;
background: linear-gradient(to right, #c6ffdd, #fbd786, #f7797d);</string>
  </property>
  <widget class="QDialogButtonBox" name="confirm_cancel_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="inProgressRb">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>In progress</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="failedRb">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Failed</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="completedRb">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Completed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;URW Gothic L&quot;;</string>
   </property>
   <property name="text">
    <string>Please choose the status of your seleceted task: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirm_cancel_btn</sender>
   <signal>accepted()</signal>
   <receiver>StatusForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirm_cancel_btn</sender>
   <signal>rejected()</signal>
   <receiver>StatusForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
